src/tokens/solana.tokenlist.json

```diff
@@ -16531,6 +16531,19 @@
         "website": "https://solana.lido.fi/"
       }
     },
+    {
+      "chainId": 101,
+      "address": "EbMg3VYAE9Krhndw7FuogpHNcEPkXVhtXr7mGisdeaur",
+      "symbol": "bSOL",
+      "name": "Anchor bonded token for stSOL",
```

Comment:
Should we call it “Lido Bonded SOL” (cf. “Lido Staked SOL” that we have for stSOL)? Or maybe “Lido Bonded SOL for Anchor”?)

Reply:
Yeah, I think it's better. Changed it